source/data/tier1data/features/collaboration.rst

```diff
@@ -0,0 +1,192 @@
+Currently, we distinguish between two groups: project groups and access groups.  
+
+A **project group** is created when a research group applies for a project in Tier-1 Data.
+It includes all collaborators on the project, some of which are **responsibles**, and has a shared collection.
+
+**Access groups** are subgroups of a project group and are used to assign specific permissions to certain users based on their roles.
+For example, data providers and data analysts may need different permissions on the datasets of the project. By creating an access group for data providers
+with the users that have this role you can define which permissions all data providers have based on that role (e.g. writing data to a certain collection).
+The same goes for data analysts, who would need read-only access to the datasets but write-access to a collection for results.
+A user that has both roles will have both sets of permissions. This way, permissions are not assigned to individuals based on who they are but based on their role in the project.
+
+For example:
+
+- Chemistry (project group)  
+    - Mary
+    - John
+    - Chris
+- Chemistry_data_providers (access group)  
+    - John
+- Chemistry_data_analysts (access group)  
+    - Mary
+    - Chris 
+
+Permissions in Tier-1 Data can be given to either individual users or groups.
+
+In the ManGO portal, permissions can only be applied at the group level.
+
+***********
+Permissions
+***********
+
+In Tier-1 Data, you can apply permissions to both data objects and collection.  
+A permission gives a group or user a certain type of access.
+
+There are 4 types of access: null (no permissions), read, write, own. As the table below shows,
+these permissions are cumulative: write permissions imply read permissions, and own permissions imply write permissions.
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Action
+     - Read
+     - Write
+     - Own
+   * - View
+     - ✓
+     - ✓
+     - ✓
+   * - Download (data object)
+     - ✓
+     - ✓
+     - ✓
+   * - Copy
+     - ✓
+     - ✓
+     - ✓
+   * - Edit/overwrite (data object)
+     -
+     - ✓
+     - ✓
+   * - Create new files/collections (collection)
+     -
+     - ✓
+     - ✓
+   * - Metadata
+     - View
+     - Edit
+     - Edit
+   * - Rename
+     - 
+     - 
+     - ✓
+   * - Move
+     - 
+     - 
+     - ✓
+   * - Delete
+     - 
+     - 
+     - ✓
+   * - Change permissions
+     -
+     -
+     - ✓
+
+
+'Null' access allows no actions. Assigning 'null' permissions is equivalent to removing existing permissions. 
+    
+Each permission relates to a specific collection or data object.  
+For example, you can give a group access to a specific collection, but not to the subcollections or data objects under it.
+
+- Chemistry (Mary - read)
+    - ExperimentA (Mary - write)
+        - result1.txt 
+        - result2.txt 
+        - result3.txt
+    - ExperimentB
+        - result1.txt 
+        - result2.txt
+        - result3.txt
```

Comment:
Fix formatting:
```suggestion
- Chemistry (Mary - read)
  - ExperimentA (Mary - write)
    - result1.txt 
    - result2.txt 
    - result3.txt
  - ExperimentB
    - result1.txt 
    - result2.txt
    - result3.txt
```

Reply:
It seems that swithcing from 4 to 2 spaces creates an 'unexpected indentation' error for list with hyphens.
I tried doing it with asterisks instead, but couldn't create a nested list that way.

Reply:
I forgot to add blank lines between the indented blocks. Sorry for the confusion. I pushed the fix to your branch in https://github.com/hpcleuven/VscDocumentation/pull/349/commits/adb42f02ea257dbf03ff030dcc41c1446147c8da